Makefile

```diff
@@ -11,7 +11,7 @@ help:
 	@echo "dev - install theblues in develop mode"
 	@echo "lint - check style with flake8"
 	@echo "test - run tests quickly with the default Python"
-	@echo "check - run tests against 2.7 and 3.4 and check lint."
+	@echo "check - clean env, run tests against 2.7 and 3.4, check lint,"
```

Comment:
That's weird--the actual file reads "clean env, run tests against 2.7 and 3.4, check lint and test docs." It does have a trailing period, which I've removed. Not sure why the diff is clipping that last part.